pkg/sos/server.go

```diff
@@ -135,7 +136,14 @@ func (s SosServer) redirectToResourceHandle(w http.ResponseWriter, r *http.Reque
 
 func (s SosServer) displaySosHandle(w http.ResponseWriter, r *http.Request) {
```

Comment:
how does this work for the wifi program, if it has its own thing it wants to read?

Let's go with this for now but we need to think on it a bit later.